packages/runtime/src/next/client.ts

```diff
@@ -134,13 +136,15 @@ export class Makeswift {
   private apiOrigin: URL
   private graphqlClient: GraphQLClient
   private runtime: ReactRuntime
-  private siteVersion: MakeswiftSiteVersion | null
+  private features: MakeswiftFeatures
+  private siteVersion: MakeswiftSiteVersion
 
   constructor(
     apiKey: string,
     {
       apiOrigin = 'https://api.makeswift.com',
       runtime = ReactRuntime,
+      features,
```

Comment:
I don't think I want to nest our features under a `features` field, I would rather have top-level configuration options